internal/data/daily.go

```diff
@@ -18,31 +19,36 @@ type DailyDownload struct {
 	Downloads int    `json:"downloads"`
 }
 
-func GetDaily(period, pkg string) (Daily, error) {
+func GetDaily(period Period, pkgs []string) ([]Daily, error) {
 	// weirdly the API returns an array of length 1
 	var daily []Daily
-
-	path := URL + "downloads/daily/" + period + "/" + pkg
-
+	path := URL + "downloads/daily/" + string(period) + "/" + strings.Join(pkgs, ",")
 	resp, err := http.Get(path)
 
 	if err != nil {
-		return Daily{}, err
+		return daily, err
+	}
+	// This is currently kind of meaningless until
+	// https://github.com/r-hub/cranlogs.app/issues/41 is resolved but at least
+	// its there
+	if resp.StatusCode != 200 {
+		return daily, err
```

Comment:
this is.... bare minimum ok - ideally would consider more about what alternative responses could come in then handle them. One huge issue is essentially this will currently swallow the result that isn't an error. So for example, the boy json with the error message currently is being swallowed - so this needs to get refactored to likely generate a better, processed error.